lib/elixir/pages/Writing Documentation.md

```diff
@@ -51,6 +51,28 @@ The compiler will infer this argument as `map`. Sometimes the inference will be
       size
     end
 
+## Documentation metadata
+
+Elixir allows developers to attach random metadata to the documentation too. This is done by using the relevant attribute, such as `@moduledoc`, `@typedoc` and `@doc` and passing a keyword list where the key is the metadata key and value is the metadata value. A commonly used metadata is `:since`, which annotates which version that particular module, function, type or callback was added. For instance, in the example above we could have added:
+
+    defmodule MyApp.Hello do
+      ...
+      @moduledoc since: "1.0.0"
+      
+      ...
+      @doc since: "1.3.0"
+    end
+
+Another existing metadata is `:deprecated`, which emits a warning in the documentation that usage of the function is discouraged:
+
+    @doc deprecated: "Use Foo.bar/2 instead"
+
+Note the `:deprecated` does not warn when a developer invokes the functions. If you want the code to also emit a warning, you can use the `@deprecated` attribute:
+
+    @deprecated "Use Foo.bar/2 instead"
+
+Metadata can have any key and is often by documentation tools to provide more accessible ways to traverse your project's documentation.
```

Comment:
```suggestion
Metadata can have any key and is often used by documentation tools to provide more accessible ways to traverse your project's documentation.
```

And maybe the first part can be moved to the top, to something like:

> Elixir allows developers to attach random metadata to the documentation too. This is done by using the relevant attribute (such as `@moduledoc`, `@typedoc`, and `@doc`) and passing a keyword list where the key is the metadata key and the value is the metadata value:
>
> ```elixir
> @doc "Returns the meaning of life"
> @doc since: "1.0.1", authors: ["John Doe"]
> def meaning_of_life, do: 42
> ```
>
> A commonly used metadata is `:since`, which annotates in which version that particular module, function, type, or callback was added. For instance, in the example above we could have added:
>
> ...